src/ophyd_async/epics/areadetector/writers/nd_file_hdf.py

```diff
@@ -37,4 +37,7 @@ def __init__(self, prefix: str, name="") -> None:
         self.capture = epics_signal_rw_rbv(bool, prefix + "Capture")
         self.flush_now = epics_signal_rw(bool, prefix + "FlushNow")
         self.xml_file_name = epics_signal_rw_rbv(str, prefix + "XMLFileName")
+        self.array_size0 = epics_signal_r(int, prefix + "ArraySize0")
+        self.array_size1 = epics_signal_r(int, prefix + "ArraySize1")
```

Comment:
A note for the future - we may need all 3 dimensions in general (width, height, depth). Consider adding `ArraySize2`.

Reply:
https://github.com/bluesky/ophyd-async/issues/454 to address it later.